course/include/isometry.h

```diff
@@ -0,0 +1,78 @@
+#ifndef ISOMETRY_H
+#define ISOMETRY_H
+
+#include <cstdint>
+#include <iostream>
+#include <ostream>
+
+namespace ekumen {
+namespace math {
+
+class Vector3{
+    public:
+        // Constructors
+        Vector3() : x_(0), y_(0), z_(0) {};
+        Vector3(double x, double y = 0, double z = 0) : x_(x), y_(y), z_(z) {};
+        Vector3(const Vector3& v) : x_(v.x()), y_(v.y()), z_(v.z()) {} ;
+
+        // Destructor
+        ~Vector3() {}; // is this necessary?
```

Comment:
Nop, it's not unless you need to do something during destruction. Otherwise the default destructor is ok.